vscode/microsoft-kiota/src/workspaceTreeProvider.ts

```diff
@@ -0,0 +1,58 @@
+import * as vscode from 'vscode';
+import * as path from 'path';
+import * as fs from 'fs';
+
+const workspaceJsonPath = path.join(vscode.workspace.workspaceFolders?.map(folder => folder.uri.fsPath).join('') || '', '.kiota', 'workspace.json');
+
+export class WorkspaceTreeProvider implements vscode.TreeDataProvider<vscode.TreeItem> {
+
+    constructor(private context: vscode.ExtensionContext) {
+        void this.ensureKiotaDirectory();
+    }
+    async getChildren(element?: vscode.TreeItem): Promise<vscode.TreeItem[]> {
+        if (!element) {
+            return [new vscode.TreeItem('workspace.json', vscode.TreeItemCollapsibleState.None)];
+        }
+        return [];
+    }
+
+    getTreeItem(element: vscode.TreeItem): vscode.TreeItem {
+        if (element) {
+            element.command = { command: 'kiota.workspace.openWorkspaceFile', title: "Open File", arguments: [vscode.Uri.file(workspaceJsonPath)], };
+            element.contextValue = 'file';
+        }
+        return element;
+    }
+
+    private async ensureKiotaDirectory(): Promise<unknown> {
+        if (!vscode.workspace.workspaceFolders || vscode.workspace.workspaceFolders.length === 0) {
+            await vscode.window.showErrorMessage(
+                vscode.l10n.t("No workspace folder found, open a folder first")
```

Comment:
missing from the bundle

Reply:
this was already there

Reply:
correct, and the translation is already present. disregard